src/Emitter/SapiStreamEmitter.php

```diff
@@ -78,10 +81,11 @@ private function emitBody(ResponseInterface $response) : void
 
     /**
      * Emit a range of the message body.
+     * @psalm-param ParsedRangeType $range
      */
     private function emitBodyRange(array $range, ResponseInterface $response) : void
     {
-        list($unit, $first, $last, $length) = $range;
+        list($_, $first, $last, $_) = $range;
```

Comment:
I don't think we need `$_` here.

```suggestion
        [, $first, $last] = $range;
```

Reply:
Oh yah, haven't really seen that it is `list`. Just wanted to fix the unused variable issue 😂